### ISSUE TITLE
Using "nodejs10.x" as runtime in config causes 502 BAD_GATEWAY error

### ISSUE THREAD

**Issue by funkybunky** [thgh/now-sapper#11]
In the readme the following is stated:
"You can change the AWS lambda runtime from the default node 8
Example `now.json`"
```
{
  "version": 2,
  "builds": [
    {
      "src": "package.json",
      "use": "now-sapper",
      "config": {
        "runtime": "nodejs10.x"
      }
    }
  ]
}
```
But when I add the nodejs 10.x runtime, deploy with `now` and access the site online, I get an "502: BAD_GATEWAY" error with "Code: NO_STATUS_CODE_FROM_FUNCTION"

I found out that in the demo repo here: https://github.com/beyonk-adventures/now-sapper-demo
there is no runtime config added. After removing it, everything worked.

Just stating this in case anybody else runs into this and to maybe update the docs.

**Comment by ryanditjia** [comment]
I’m also getting the 502 BAD GATEWAY error when setting package.json engines to 12.x.

**Comment by ajbouh** [comment]
I made a fork that fixed the problem for me: https://github.com/ajbouh/now-sapper/

**Comment by silasb** [comment]
@ajbouh can you issue a PR then?

**Comment by antony** [comment]
@ajbouh does your fork work?

**Comment by antony** [comment]
Just tested it. It works - and works well with v12 too. I'll make the required modifications and open a PR.

**Comment by ajbouh** [comment]
Awesome, thanks! I saw that it was a merge conflict and haven't gotten a chance to cycle back.

**Comment by antony** [comment]
This now works thanks to @ajbouh :champagne:

**Comment by thgh** [comment]
Thanks @ajbouh and @antony, this has been released in v0.38.0

In the next weeks I will be working on a sapper project that's hosted on Now, so looking forward to enjoy this fix :-)
